tests/test_organizations.py

```diff
@@ -147,13 +151,15 @@ def test_update_organization_with_domain_data(
 
         assert updated_organization.id == "org_01EHT88Z8J8795GZNQ4ZP1J81T"
         assert updated_organization.name == "Example Organization"
-        assert updated_organization.domains == [
-            {
-                "domain": "example.io",
-                "object": "organization_domain",
-                "id": "org_domain_01EHT88Z8WZEFWYPM6EC9BX2R8",
-            }
-        ]
+        assert updated_organization.domains[0].dict() == {
+            "domain": "example.io",
+            "object": "organization_domain",
+            "id": "org_domain_01EHT88Z8WZEFWYPM6EC9BX2R8",
+            "state": "verified",
+            "organization_id": "org_01EHT88Z8J8795GZNQ4ZP1J81T",
+            "verification_strategy": "dns",
+            "verification_token": "token",
+        }
```

Comment:
What precipitated this change?

Reply:
Me typing Organization -> domains as `OrganizationDomain` instead of a dict.

Reply:
Ah, yup. Our SDK should return models. 👍